t/t2801-top-cmd.t

```diff
@@ -79,11 +79,15 @@ test_expect_success 'submit batch script and wait for it to start' '
 	cat >batch.sh <<-EOT &&
 	#!/bin/sh
 	touch job2-has-started
-	flux mini run sleep 300
+	jobid=\$(flux mini submit --flags waitable sleep 300)
+	flux job wait-event \$jobid start
+	touch job2-batch-sleep-has-started
+	flux job wait --all
 	EOT
 	chmod +x batch.sh &&
 	flux mini batch -t30m -n1 batch.sh >jobid2 &&
-	$waitfile job2-has-started
+	$waitfile job2-has-started &&
+	$waitfile job2-batch-sleep-has-started
```

Comment:
It would probably be simpler to just wait for `job2-batch-sleep-has-started` (or just move `job2-has-started` to after `flux job wait-event` above), since appearance of `job2-batch-sleep-has-started` implies `job2-has-started`...